.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - 2.7
+before_install:
+  - export DISPLAY=:99
+  - sh -e /etc/init.d/xvfb start
+install:
+  - pip install -r test-requirements.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/base.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/test.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock/requirements.txt
```

Comment:
@nedbat Our integration tests on Travis CI use bok choy and selenium to run through the various XBlock views/handlers hosted within the XBlock SDK "workbench" runtime. That means that we need to install XBlock, XBlock SDK, bok choy, mock, ddt, etc. But if we specify those requirements in our own XBlock's requirements.txt (whether using pinned versions or not), we'll inevitably arrive at a situation where our XBlock requires a different version of some dependency than xblock-sdk does, which leads to inconsistent test results depending on environment and order of installation. And for people like me who have an "xblock venv" where I install xblock-sdk plus about 20 different XBlocks that I work on, version mismatches can be frustrating time sinks.

So we've found that the easiest solution for now is just to say "we'll have what they're having" with respect to xblock-sdk, and standardize on whatever requirements it is using.